templates/opps-list-widget.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * This template is used to display the list of flexible volunteer opportunities
+ * for the widget.
+ *
+ * To adjust this template copy it into your current theme within a folder called "wivm".
+ */
+
+// Global variables that are set in widget/class-widget.php via the widget settings in admin 
```

Comment:
This is a tough one with no clear solution. I guess if we're going to use a global my only suggestion would be to use one variable and make it an array. That way we only have one global variable instead of four. 

Otherwise, I don't have any great suggestions on this one.